backend/app/libs/series.py

```diff
@@ -24,4 +24,4 @@ def make_hist_price_series(
         (price.value for price in prices),
         index=pd.Index((price.timestamp for price in prices), name="timestamp"),
         name=f"{token_symbol} historical price",
-    )
+    ).sort_index()
```

Comment:
Just for my knowledge, did you notice that the indices were not sorted before introducing this `.sort_index()`?

Reply:
Yes, they were sorted in descending order for `Prices`.

The biggest consequence being that the `std_dev` calculation was incorrect.

Reply:
Here is the summary representation of a given price dataframe before and after the change:

Before:
<img width="435" alt="Screenshot 2022-07-29 at 14 17 30" src="https://user-images.githubusercontent.com/1884912/181769534-8b95ca04-22b3-46db-b9d0-2337b4d84180.png">


After:
<img width="353" alt="Screenshot 2022-07-29 at 14 19 20" src="https://user-images.githubusercontent.com/1884912/181769520-9a40a88e-0de3-410f-9509-ce61ca918f7d.png">

Indeed the std_dev calcuation, albeit being very similar, is not the same. Nevertheless, I guess the volatility calculation stays the same.

This index-sorting fix has fairly wide-ranging consequences. We might need to look into git history to look for when a bug was introduced (think git bisect). A good rule of thumb is thus to always move such a fix to a separate independent commit.